### DISCUSSION TITLE
Trying to update packages overwrites package.json

### DISCUSSION THREAD

**Discussion by krichards** [pnpm/pnpm#6626] [Q&A]
So we have a package which is at ^0.28.5  and we would like to keep getting updates up to v1 is released, whilst we can set the package.json to <1.0.0, when update is run the package.json is then modified to be ^0.28.5 again, and we are back stuck with not getting updates until we manually change the package.json again.

Any suggestions on how we can get pnpm to give us updates to a package which has not been released?

**Comment by ghost** [comment]
I do not think so at the moment. Both `install` and `update` modify `package.json` and `pnpm-lock.json`, to match the latest installed version. I'm not sure what the rational behind this is. I, for one, would prefere a to have a way to leave `package.json` as is (there might be elaborate rules in there, to avoid unusable combinations of dependencies) and only update the `pnpm-lock.json` and/or `node_modules`.
